### PR TITLE
Simplify local development

### PR DESCRIPTION
**Pull request by MatiasOS**
This is the first approach to run dev server. Let me know if you can run it easily. 
Suggestions are very welcome!

### CODE REVIEW

File: src/local-network.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  const accounts = await web3.eth.getAccounts();
+  const { address } = await contract.new({
+    from: accounts[0],
+    gas: 6000000,
+  });
+  config.set('indexAddress', address);
+  console.log(`WTIndex at ${address}!`);
+};
+
+const transferFounds = async () => {
```
Comment (JirkaChadima) [inline src/local-network.js]: This is supposed to be named `transferFunds`, there's a typo

File: src/local-network.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+const getContractWithProvider = (metadata, provider) => {
+  let contract = new TruffleContract(metadata);
+  contract.setProvider(provider);
+  contract = hackInSendAsync(contract);
+  return contract;
+};
+
+const deployIndex = async () => {
+  const contract = getContractWithProvider(WTIndexContract, provider);
```
Comment (JirkaChadima) [inline src/local-network.js]: I'd rename the `contract` const to be more specific, such as `indexContract`

File: README.md [hunk 3]
```diff
@@ -16,6 +16,43 @@ npm install
...
+### Running dev mode
+With all the dependencies installed, you can start the dev server
+
+First step is add the accounts to the `configuration.json` file. This accounts
+will receive some Ether from ganache accounts.
+```javascript
+{
+  local : {
+    ...
+    "users": ['0x0...1', '0x0...2']
```
Comment (JirkaChadima) [inline README.md]: What about pre-populating the ganache-cli with some demo accounts during startup? Like we have for tests. In that way, the `configuration.json` would not have to be modified at all. And in demo app we have a pre-initialized wallet as well.

File: README.md [hunk 4]
```diff
@@ -16,6 +16,43 @@ npm install
...
+
+```
+Now, we can start Ganache client and run dev server
+
+```bash
+npm run dev-net # Start Ganache client
+npm run dev # Start server with ETH_NETWORK=local.
+```
+When `ETH_NETWORK=local` we run internally a script to deploy WT Index and
+transfer founds to the user account.
```
Comment (JirkaChadima) [inline README.md]: `founds` -> `funds`

File: README.md [hunk 5]
```diff
@@ -16,6 +16,43 @@ npm install
...
+```javascript
+{
+  "schemes": ["http"],
+  "host": "localhost:3000"
+}
+
+```
+
+
 ## Examples
```
Comment (JirkaChadima) [inline README.md]: I think this section is slightly obsolete. I forgot to update it when I was adding support for multiple wallets. Should we updated as part of this PR?

File: src/local-network.js (hunk 1 above)
Comment (MatiasOS) [reply src/local-network.js]: I have changed this function in favor of default accounts

File: src/local-network.js (hunk 2 above)
Comment (MatiasOS) [reply src/local-network.js]: done

File: README.md (hunk 5 above)
Comment (MatiasOS) [reply README.md]: Updated!

File: src/local-network.js (hunk 2 above)
Comment (MatiasOS) [reply src/local-network.js]: Done

File: README.md (hunk 3 above)
Comment (MatiasOS) [reply README.md]: Now, by default we have 2 accounts. Now we have 2 accounts. One is encrypted in `/keys/`

File: README.md [hunk 6]
```diff
@@ -38,7 +38,7 @@ npm run dev-net # Start Ganache client
 npm run dev # Start server with ETH_NETWORK=local.
 ```
 When `ETH_NETWORK=local` we run internally a script to deploy WT Index and
-transfer founds to the user account.
+transfer fonds to the user account.
```
Comment (JirkaChadima) [inline README.md]: Another typo :)

File: README.md [hunk 7]
```diff
@@ -17,28 +17,22 @@ npm test
...
-}
-
+With all the dependencies installed, you can start the dev server.
+First step is start Ganache.
+```bash
+npm run dev-net
 ```
-Now, we can start Ganache client and run dev server
 
+Gache starts with 2 default accounts, witch one is an hotel manager with address `0x0ba3cd50b07ee204a47246b6b2f274fd41805c47`.
```
Comment (JirkaChadima) [inline README.md]: Gache -> Ganache; witch one -> one of them

**Comment by kvbik** [comment]
lot of stuff going on here. what  is  missing to merge it @MatiasOS ?

**Pull request merged**